src/scripts/fetcher/TachiyomiAZ.js

```diff
@@ -68,10 +68,14 @@ export class TachiyomiAZ extends BaseVariant {
 	};
 
 	stripPreviewData = data => {
+		var preview_download = {
+			browser_download_url: TACHIYOMI_AZ_PREVIEW_API_URL,
+		};
+		var version_release = new Date(data * 1000).toLocaleDateString("en-US");
```

Comment:
Use Camelcase and move `preview_download` into the return body
```suggestion
		var versionRelease = new Date(data * 1000).toLocaleDateString("en-US");
```

Reply:
weird, that's exactly what I did before and it kept not being able to download. Works locally now!